helpers/mathjax.js

```diff
@@ -1,5 +1,21 @@
 let mathJaxLoaded;
 
```

Comment:
Could you see the MathJax renderer moving into the LMS like the others? Might have to do something for iFRAs, but I think they'll need to be handled either way for the other renderers.

Reply:
Hmmm, I think it could but yeah, I think it would introduce some difficulties with iFRAs, and also math wouldn't work in app component demos.  I think we would also like to use MathJax to render math in the HtmlEditor's design-view if possible, but it remains to be seen if that's doable, and also what we'd need specifically.  Ultimately we would like a single loader and configuration for MathJax, since unfortunately we've seen some apps go on their own with it, which is now causing us some issues.  I could see perhaps moving the MathJax loader and renderer into its own repo if we didn't want it part of core.  In the spirit of that, we will likely have a prismjs loader/renderer as well.

Reply:
Gotcha cool that makes sense. Just trying to figure out where we'd draw the line between putting something in core vs. an LMS plugin.